css/menu.css

```diff
@@ -15,6 +15,16 @@
   padding-top: 44px;
 }
 
+html.web .fl-with-top-menu main.fl-page-content-wrapper,
+html.android .fl-with-top-menu main.fl-page-content-wrapper {
+  /* Remove the height of the menu */
+  min-height: calc(100vh - 44px);
+}
+
+html.web[data-has-notch].supports-container .fl-with-top-menu {
```

Comment:
You were quite deliberate about removing `.web` in other PRs to only use `[data-has-notch]`. Now you're using `html.web` as part of the selector too. Be consistent.